test/nn/test_linear_module_validation.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import torch
+from torch.nn import Parameter
+from torch.nn.modules.linear import Linear, Bilinear
+
+class TestLinearValidation(unittest.TestCase):
```

Comment:
these can be added as [`ErrorModuleInput`](https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_modules.py#L188)s for `Linear` and `Bilinear`, [see usage here for an example](https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_modules.py#L4215)

Reply:
Thans for the suggestion. I haved added them accordingly